### ISSUE TITLE
printf error in CCTM.php, line 1515

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#47]
```
Warning message on plugin activation:

Warning: printf() [function.printf]: Argument number must be greater than zero 
in 
/nfs/c03/h06/mnt/57308/domains/integralcommunity.ca/html/wp-content/plugins/cust
om-content-type-manager/includes/CCTM.php on line 1522

What steps will reproduce the problem?
1. Install Custom Content Type Manager plugin version 0.8.6.
2. Activate plugin from the WordPress dashboard Plugins page

The script error that causes this is actually 1515. The offending PHP statement 
is:
________________

printf('<div id="custom-post-type-manager-warning" class="error">
    <p>
        <strong>%$1s</strong>
        <ul style="margin-left:30px;">
            %2$s
        </ul>
    </p>
    </div>'
    , $msg
    , $error_items);
________________

The error occurs because of a typo in the first printf format parameter: "%$1s" 
should be "%1$s"
```

Original issue reported on code.google.com by `paul.nov...@gmail.com` on 10 Mar 2011 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
Yes, I believe this was fixed in 0.8.7, but sorry, the WP plugin repo screwed 
the pooch on this one. 
```

Original comment by `fireproofsocks` on 10 Mar 2011 at 7:49
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in version 0.8.8.
```

Original comment by `fireproofsocks` on 11 Apr 2011 at 9:58
- Changed state: **Fixed**
